google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -823,6 +836,13 @@ QueryOptions getQueryOptions() {
     return queryOptions;
   }
 
+  /**
+   * The initial returnCommitStats value for connections created by this {@link ConnectionOptions}
```

Comment:
Only the initial value? If it changes, this still returns the initial value? That's surprising. Hmm, looks like it's final so no need to say initial. probably rewrite as, "whether connections created by this {@link ConnectionOptions} return commit stats"